src/domain/usecases/ValidateGlassModulesUseCase.ts

```diff
@@ -8,16 +8,28 @@ const glassModules = [
         id: "AVnpk4xiXGG",
```

Comment:
Not part of this PR, but as note for your team for the future: we shouldn't hardcode ID's. Typically, we'd have a structure containing all the required metadata name/code, and some repository gets from the DB to get all the info for the object (which will include the ID).

Reply:
Noted in glass team slack, will be addressed in upcoming tasks.